src/node_nanomsg.cc

```diff
@@ -23,6 +23,11 @@ using v8::Object;
 using v8::String;
 using v8::Value;
 
+#define ret NanReturnValue
+#define utf8 v8::String::Utf8Value
```

Comment:
It looks like you can pass cflags arguments to node-gyp, but note sure it's safe to bet on end users having c++11 support just yet.  I wonder what @trevnorris thinks?